ocamlorg.opam

```diff
@@ -38,6 +38,9 @@ depends: [
   "yojson"
   "lwt"
   "brr"
+  "rpclib"
+  "rpclib-lwt"
+  "ppx_deriving_rpc"
```

Comment:
We need to add these to the `dune-project` deps

Reply:
Ah yes!